### ISSUE TITLE
Missing development instructions

### ISSUE THREAD

**Issue by toolmantim** [probot/probot.github.io#208]
Whilst working on #185 I wanted to spin up a local site to check the new app page would look good, but the commands to get it running locally don't seem to be documented anywhere. They were:

```shell
git submodule update --init --recursive
# Install Ruby dependencies
bundle
# Start the site on localhost:4000
bundle exec jekyll serve
# Run the lint checks
bundle exec ruby test/*_test.rb
```

I think making this info accessible would help encourage higher quality PR submissions.

Happy to submit a PR if you'd like, just let me know where it should belong (I suspect the readme, but it looks intentionally bare).

**Comment by RichardLitt** [comment]
Good catch, @toolmantim. These instructions are in the `scripts` folder. 

I think this will be resolved with https://github.com/probot/probot.github.io/pull/181.

**Comment by hiimbex** [comment]
Also wanted to note that most (?) of this is already documented in our [Conturbuting.md](https://github.com/probot/probot.github.io/blob/master/CONTRIBUTING.md#submitting-a-pull-request) and we link to that from the README.

Is there a way this could be more clear @toolmantim ?

**Comment by toolmantim** [comment]
Ah, I didn't spot that at all @hiimbex! Yeah, those instructions cover all of this, and I actually looked at the [CONTRIBUTING.md](https://github.com/probot/probot.github.io/blob/master/CONTRIBUTING.md) instructions earlier, but for the "Adding your app" section — so who knows? 🤷🏼‍♂️
